tests/viscoplastic_melt_fraction.prm

```diff
@@ -0,0 +1,122 @@
+# This test checks whether the katz2003 mantle melting model in the
+# reactive transport model produces the correct melt fraction output.
+#
+# Temperature will be held constant throughout model at 1573 K so at 1 GPa
+# melt fraction should be 15% from Katz et. al. 2003 Fig. 4
+
+set Adiabatic surface temperature          = 1573
+
+# The number of space dimensions you want to run this program in.
+set Dimension                              = 2
+
+# The end time of the simulation. Units: years if the 'Use years in output
+# instead of seconds' parameter is set; seconds otherwise.
+set End time                               = 1000
+set Pressure normalization                 = surface
+set Surface pressure                       = 0
+set Resume computation                     = false
+set Start time                             = 0
+set Use years in output instead of seconds = true
+set Output directory                       = output-vp-melt-fraction
+
+
+subsection Boundary temperature model
+  set List of model names = initial temperature
+  set Fixed temperature boundary indicators   = 0,3
```

Comment:
```suggestion
  set Fixed temperature boundary indicators   = top, bottom
```
This should not make any difference, but I find it much more readable to refer to the boundaries via name.